phoenix-core/src/it/java/org/apache/phoenix/end2end/IndexToolIT.java

```diff
@@ -821,6 +942,7 @@ public static IndexTool runIndexTool(Configuration conf, boolean directApi, bool
                 indexTableName, tenantId, verifyType, disableLoggingType);
         List<String> cmdArgList = new ArrayList<>(Arrays.asList(cmdArgs));
         cmdArgList.addAll(Arrays.asList(additionalArgs));
+        LOGGER.info("Running IndexTool with {}", Arrays.toString(cmdArgList.toArray()), new Exception("Stack Trace"));
```

Comment:
You want to keep this here?

Reply:
I found it really useful, I think it's OK for a test case.
I'm going to fix up the exception message to something more descriptive.

Reply:
Ok! If you find it useful, then that's great.